src/modules/translation/languages/en.json

```diff
@@ -206,10 +206,15 @@
     }
   },
   "create_multiple_items_modal": {
-    "title": "Create multiple items",
-    "successful_title": "Items created",
-    "cancelled_title": "Cancelled multiple items creation",
-    "failed_title": "Failed multiple items creation",
+    "create_title": "Create multiple items",
```

Comment:
I think it might be better to have another level in the object, one for the creation an the other one for the editing.
Something like:
```
"create_multiple_items_modal": {
   "create" : {
     "title": ...,
     subtitle: ...,
  },
  "edit": {
    "title": ...,
    "subtitle": ...
  }
}
```

Reply:
sure! done!